reservation_units/utils/reservation_unit_reservation_scheduler.py

```diff
@@ -80,15 +80,11 @@ def is_reservation_unit_open(self, start_datetime: datetime, end_datetime: datet
 
         return origin_hauki_resource.is_reservable(start_datetime, end_datetime)
 
-    def get_reservation_unit_possible_start_times(
-        self,
-        selected_date: date,
-        interval_minutes: int,
-    ) -> set[datetime]:
+    def get_reservation_unit_possible_start_times(self, selected_date: date, interval_minutes: int) -> set[datetime]:
         resource = self.reservation_unit.origin_hauki_resource
         reservable_time_spans: QuerySet[ReservableTimeSpan] = resource.reservable_time_spans.filter(
-            start_datetime__date=selected_date,
-            end_datetime__date=selected_date,
+            start_datetime__date__lte=selected_date,
+            end_datetime__date__gte=selected_date,
         )
```

Comment:
What was this change supposed to do? 🤔

Reply:
This function did not pick up multi-day reservable timespans if they overlapped with the selected date, leading to no possible start times being found. Found this during testing, but didn't make a specific test for it.